pkg/core/auth/config.go

```diff
@@ -0,0 +1,43 @@
+package auth
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/url"
+)
+
+// getOAUTHInfo queries the Udash website to retrieve OAUTH configuration
+func getOAUTHInfo(endpointURL string) (issuer string, audience string, clientID string, err error) {
```

Comment:
```suggestion
func getOauthInfo(endpointURL string) (issuer string, audience string, clientID string, err error) {
```

Does not seem to be an acronym, so better to stick with the usual CamelCase

Reply:
Well it's an acronym for Open Authorization, https://en.wikipedia.org/wiki/OAuth
I hesitated between `OAUTH` and `Oauth` and imho the first is more appropriate

Reply:
I'm nitpicky but it's not an acronym but a "shorthand", so the "all letters in uppercase" rule does not apply.

I haven't seen any other usage with uppercase so that's why I nitpicked on it but no porblem to keep it. I'm just wondering if there isn't a trademark such as for Apache for instance (but I'm not sure where to look for this).